src/docker-in-docker/install.sh

```diff
@@ -10,7 +10,7 @@
 
 DOCKER_VERSION="${VERSION:-"latest"}" # The Docker/Moby Engine + CLI should match in version
 USE_MOBY="${MOBY:-"true"}"
-MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION}"
+MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION:-"latest"}"
```

Comment:
We intentionally avoided to set the default value for `MOBY_BUILDX_VERSION`, the @devcontainer/cli computes it from the [devcontainer-feature.json](https://github.com/devcontainers/features/blob/main/src/docker-outside-of-docker/devcontainer-feature.json#L25)

This adds a redundancy and whenever we want to update `default`, this leads to updating in both the placed. Failing to update in one place leads to confusions (we were bit by this few times, learned from experience)

Reply:
All other env vars already have a default value and haven't been changed for a long time. Using `latest` (like in `DOCKER_VERSION`) there wouldn't be any issue as the `.json` config overrides the default value: no need to change in the `.sh`.